src/analyses/extractPthread.ml

```diff
@@ -850,7 +850,7 @@ module Codegen = struct
 
     Writer.write "promela model" "pml" promela ;
     Writer.write "graph" "dot" dot_graph ;
-    print_endline
+    Logs.info
```

Comment:
I think this a bit too important to just stick it into a `Logs.info`. It is not something to be logged but directly informs the user about what to do.